R/d.e.mcmc.R

```diff
@@ -46,9 +46,10 @@ d.e.mcmc <- function(f, inits, max.iter, n.walkers, ...) {
   p()
 
   for (l in seq_len(chain.length)[-1]) {
-    z <- 2.38 / sqrt(2 * n.params)
+    ## # See Nelson et al. (2013), eq(10): https://doi.org/10.1088/0067-0049/210/1/11/.
```

Comment:
Could we put the reference in the references section of the documentation above please? We can then drop the doi from here and resolve lints.